src/main/java/seedu/duke/PassengerParser.java

```diff
@@ -0,0 +1,16 @@
+package seedu.duke;
+
+import java.util.Arrays;
+
+public class PassengerParser {
+    private static PassengerList passengers = new PassengerList();
+
+    public static void parse(String[] inputWords) {
+        String command = inputWords[1];
+        if (command.equalsIgnoreCase("add")) {
```

Comment:
I think this can be made into a global boolean variable and rename to isAdd for easier identification.